### PR TITLE
feat: add passkey warning message for unsupported devices

### PR DESCRIPTION
**Pull request by pranalidhanavade**
### What 

- Added passkey warning messages for unsupported devices.

### CODE REVIEW

File: src/components/Authentication/SignUpUserPasskey.tsx [hunk 1]
```diff
@@ -33,15 +35,21 @@ const SignUpUserPasskey = ({ email, firstName, lastName }: { email: string, firs
...
 
     useEffect(() => {
-
         if (window?.location?.search.length > 7) {
             setEmailAutoFill(window?.location?.search.split('=')[1])
         }
-    }, [])
 
+        const platform = navigator.platform.toLowerCase();
+        console.log("platform", platform)
```
Comment (bhavanakarwade) [inline src/components/Authentication/SignUpUserPasskey.tsx:46]: Remove console

**Review by bhavanakarwade — CHANGES_REQUESTED** [review]

File: src/components/Authentication/SignInUserPasskey.tsx [hunk 2]
```diff
@@ -45,6 +46,15 @@ const SignInUserPasskey = (signInUserProps: signInUserProps) => {
 		setShowSignInUser(false);
 	};
 
+	useEffect(() => {
+        const platform = navigator.platform.toLowerCase();
+        console.log("platform", platform)
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/components/Authentication/SignInUserPasskey.tsx:51]: I have implemented that console for testing on DEV. I will remove it after testing it on DEV.

**Review by bhavanakarwade — APPROVED** [review]

File: src/components/Profile/AddPasskey.tsx [hunk 3]
```diff
@@ -189,6 +192,11 @@ const AddPasskey = ({ responseMessages }: { 	responseMessages: (value: IResponse
 		} else {
 			setProfile();
 		}
+		const platform = navigator.platform.toLowerCase();
+        console.log("platform", platform)
+        if (platform.includes('linux')) {
```
Comment (tipusinghaw) [inline src/components/Profile/AddPasskey.tsx:197]: Don't use static value create ENUM for device type

**Review by tipusinghaw — APPROVED** [review]

File: src/components/Profile/AddPasskey.tsx (hunk 3 above)
Comment (pranalidhanavade) [reply src/components/Profile/AddPasskey.tsx]: Resolved

**Pull request merged**